postgres/migrations/V1.7__reviews_contract_type_option.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE reviews
+	ADD COLUMN contractType varchar(20)	NOT NULL 
```

Comment:
A new column like this cannot be `NOT NULL` unless you also give it a default value. Also I recommend using the `text` type instead of `char` or `varchar`. There is no benefit to using  `char` or `varchar` in PostgreSQL, they just complicate things.

Reply:
Thanks for pointing that out, it has been fixed.